content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -6,11 +6,8 @@ min-kubernetes-server-version: v1.9
 content_template: templates/task
```

Comment:
Is the min-kubernetes-server-version still correct for these updates?

Reply:
yes, the minimum version in this case is 1.9 when CoreDNS was introduced as alpha:
we are removing the following text from the file - `In Kubernetes 1.9, [CoreDNS](https://coredns.io) is available as an alpha feature`